src/main/java/nl/tudelft/ti2206/group9/level/Track.java

```diff
@@ -111,6 +117,53 @@ public final Player getPlayer() {
 	}
 
 	/**
+	 * @param amount the amount of distance to add
+	 */
+	void addDistance(final int amount) {
+		distance += amount;
+	}	
+	
+	/**
+	 * @return the distance
+	 */
+	int getDistance() {
+		return distance;
+	}
+
+	/**
+	 * @param dist the distance to set
+	 */
+	void setDistance(final int dist) {
+		Track.distance = dist;
+	}
+		
+	/**
+	 * Updates the current distance every 50 moves.
+	 * @return updated distance
+	 */
+	public int moduloDistance() {
+		if ((distance % 50) == 0) {
```

Comment:
Magic number; better extract "50" to a constant (then you can also use this constant in the test suite and other places if needed).

Also, why not just `return ((int) distance) % 50;` ? Then you don't have to keep updating the `modDistance` field :)